README.md

```diff
@@ -6,7 +6,7 @@ need. The unqualified is a JDK that also includes Maven.
 
 ## Release process
 The Docker build is driven by `--build-arg java_version`. The value of this must be Alpine's
-most specific Java 15 version, ex `15.0.1_p9-r0`.
+most specific Java 15 version without the revision classifier, ex `15.0.1_p9-r0` -> `15.0.1_p9`
```

Comment:
This keeps the tag JDK version semantic. the 'r' classifier is about the APK packaging and almost always zero